spec/model/project_spec.rb

```diff
@@ -74,4 +74,32 @@
       expect(project.default_location).to eq Option.locations.first.name
     end
   end
+
+  it "calculates current resource usage" do
```

Comment:
Could you add a test to confirm that all quota IDs are unique, similar to what we do in billing rates?